src/c-writer.cc

```diff
@@ -807,6 +820,12 @@ void CWriter::Write(TypeEnum type) {
     case Type::I64: Write("WASM_RT_I64"); break;
     case Type::F32: Write("WASM_RT_F32"); break;
     case Type::F64: Write("WASM_RT_F64"); break;
+    case Type::FuncRef:
+      Write("WASM_RT_FUNCREF");
+      break;
+    case Type::ExternRef:
+      Write("WASM_RT_EXTERNREF");
+      break;
```

Comment:
Ditto single line cases.

Reply:
Same -- the linter seems to force this formatting for newly added lines of code. :-/

Reply:
Done (same as above)